### ISSUE TITLE
svelte version bump?

### ISSUE THREAD

**Issue by btakita** [rollup/rollup-plugin-svelte#18]
Is it time to bump the svelte dependency to the current version? (1.30.0 currently)

**Comment by Rich-Harris** [comment]
Nah — it's semver, it'll always get the latest 1.x version. If you need to upgrade Svelte in an existing project, you should be able to do that by installing it separately with `npm i svelte@latest`.

**Comment by btakita** [comment]
I got it to work on my development machine by running `npm i svelte@latest` && `yarn` (or `npm install`).

I attempted a deployment on Heroku & it install a custom svelte version `1.25.0`.

To fix the deployment, I manually edited the version of svelte under the `rollup-plugin-svelte@^3.1.0` entry to:

```
rollup-plugin-svelte@^3.1.0:
  version "3.1.0"
  resolved "https://registry.yarnpkg.com/rollup-plugin-svelte/-/rollup-plugin-svelte-3.1.0.tgz#12405b2ac51016c1a35cc1be6840ac0ad6df8163"
  dependencies:
    require-relative "^0.8.7"
    rollup-pluginutils "^2.0.1"
    sourcemap-codec "^1.3.1"
    svelte "^1.41.2"
```

Not sure why it's not operating correctly for deployments, but for now, I can edit `yarn.lock` until a better fix is found.

On a side note, an unfortunate side effect of upgrading svelte is the transpile time has increased from about 1.5 min to 4.5min.
